grades/models.py

```diff
@@ -16,27 +16,45 @@ class Course (models.Model):
         ]
     )
 
+    @classmethod
+    def addCourse(cls, course_name, credinitials):
+        newCourse = Course()
```

Comment:
Your course model now has a user attached to it as well.
This is a change @or313 made in PR #67